### PR TITLE
Feat/cli upgrades

### PR DESCRIPTION
**Pull request by znat**
CLI suggests npm updates and project upgrades

I made sure to
- [ ] write tests for the feature
- [ ] run all tests
- [ ] document the feature if needed

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: cli/src/commands/init.js [hunk 1]
```diff
@@ -11,12 +11,13 @@ import inquirer from 'inquirer';
...
 import { dockerComposeUp } from './services';
 import { uniqueNamesGenerator } from 'unique-names-generator';
-import { getServices, updateProjectFile, generateDockerCompose, failSpinner, startSpinner, succeedSpinner, verifySystem, consoleError, stopSpinner, getMissingImgs, getContainerNames } from '../utils';
+import { getServices, updateProjectFile, generateDockerCompose, failSpinner, startSpinner, succeedSpinner, verifySystem, consoleError, stopSpinner, getMissingImgs, getContainerNames, displayUpdateMessage } from '../utils';
 
 const access = promisify(fs.access);
 const copy = promisify(ncp);
 
 export async function initCommand(cmd) {
+    await displayUpdateMessage();
```
Comment (pheel) [inline cli/src/commands/init.js]: Don't know if there's a way using commander js to call displayUpdateMessage first regardless of the command being executed.

File: cli/src/utils.js [hunk 2]
```diff
@@ -73,6 +73,28 @@ export function stopSpinner(spinner) {
     }
 }
 
+export async function getLatestVersion() {
+    try {
+        const response = await axios.get('https://registry.npmjs.org/botfront');
+        if (response.status === 200) {
```
Comment (pheel) [inline cli/src/utils.js]: have this without `if` block

File: cli/src/utils.js [hunk 3]
```diff
@@ -73,6 +73,28 @@ export function stopSpinner(spinner) {
...
+            return response.data['dist-tags'].latest;
```
Comment (pheel) [inline cli/src/utils.js]: have this without `if` block

File: cli/src/utils.js [hunk 4]
```diff
@@ -73,6 +73,28 @@ export function stopSpinner(spinner) {
...
+        }
+        // don't do anything in case of error, just no update message
+    } catch(e) {}
```
Comment (pheel) [inline cli/src/utils.js]: return falsy value in catch block

File: cli/src/utils.js [hunk 5]
```diff
@@ -73,6 +73,28 @@ export function stopSpinner(spinner) {
...
+}
+
+export async function shouldUpdate() {
+    const currentVersion = getBotfrontVersion();
+    const latestVersion = await getLatestVersion();
+    return latestVersion !== currentVersion;
```
Comment (pheel) [inline cli/src/utils.js]: if latestVersion is falsy, return false, and maybe display message about not being able to fetch latest version tag

File: cli/src/commands/init.js (hunk 1 above)
Comment (znat) [reply cli/src/commands/init.js]: I don't think we need that. init and up should be sufficient

File: cli/src/utils.js (hunk 5 above)
Comment (znat) [reply cli/src/utils.js]: should be fixed with changes in `getLatestVersion()`

File: cli/src/commands/init.js (hunk 1 above)
Comment (pheel) [reply cli/src/commands/init.js]: ok

**Pull request merged**